kubernetes/client/ws_client.py

```diff
@@ -191,6 +191,14 @@ def run_forever(self, timeout=None):
             while self.is_open():
                 self.update(timeout=None)
 
+    def close(self, **kwargs):
```

Comment:
Look at WebScoket close method for more information: https://github.com/websocket-client/websocket-client/blob/master/websocket/_core.py#L372
that method does not return any status.